CryptoTechReminderSystem.Test/HarvestGatewayTests.cs

```diff
@@ -13,117 +12,136 @@ public class HarvestGatewayTests
     {
         private const string Address = "http://localhost:8050/";
         private const string Token = "xxxx-xxxxxxxxx-xxxx";
-        private FluentSimulator _fluentSimulator;
-        private HarvestGateway _harvestGateway;
 
-        [SetUp]
-        public void Setup()
+        [TestFixture]
+        public class CanRequestDevelopers
         {
-            _fluentSimulator = new FluentSimulator(Address);
-            _fluentSimulator.Start();
-            _harvestGateway = new HarvestGateway(Address, Token);
-        }
-        
-        [TearDown]
-        public void TearDown()
-        {
-            _fluentSimulator.Stop();
-        }
-
-        private void SetUpUsersApiEndpoint(string id, string firstName, string lastName, string email)
-        {
-            var json = $"{{  \"users\":[    {{      \"id\":{id},      \"first_name\":\"{firstName}\"" +
-                       $",      \"last_name\":\"{lastName}\",      \"email\":\"{email}\"    }}  ]}}";
-            _fluentSimulator.Get("/api/v2/users").Responds(json);
-        }
-        
-        private void SetUpUsersTimeSheetApiEndpoint(string jsonFilePath)
-        {
-             var json = File.ReadAllText(
-                 Path.Combine(
-                     AppDomain.CurrentDomain.BaseDirectory,
-                     jsonFilePath
-                 )
-             );
-            _fluentSimulator.Get("/api/v2/time_entries").Responds(json);
-        }
-       
-        
-        [Test]
-        public void CanGetDeveloper()
-        {
-            SetUpUsersApiEndpoint(
-                "123",
-                "Wen Ting",
-                "Wang",
-                "ting@email.com"
-            );
+            private FluentSimulator _fluentSimulator;
```

Comment:
I think we should rename this to `harvestApi` like how we've done in the acceptance test.

Reply:
Resolved in https://github.com/chubberlisk/crypto_tech/pull/14/commits/75cfa6f2ab7519f8774e13d0dbef8f96f80b0a73